tests/sqllogictests/suites/ee/04_ee_inverted_index/04_0000_inverted_index_base.test

```diff
@@ -217,10 +217,163 @@ Filter
+(20, 'CockroachDB: The Definitive Guide', 'Guy Harrison, Jesse Seldess, Ben Darnell', 'Get the lowdown on CockroachDB, the distributed SQL database built to handle the demands of today’s data-driven cloud applications. In this hands-on guide, software developers, architects, and DevOps/SRE teams will learn how to use CockroachDB to create applications that scale elastically and provide seamless delivery for end users while remaining indestructible. Teams will also learn how to migrate existing applications to CockroachDB’s performant, cloud-native data architecture.')
+
+query IFT
+SELECT id, score(), title FROM books WHERE match('title^5, description^1.2', 'python') ORDER BY score() DESC
+----
+2 8.500097 Python深度学习（第2版）
+6 6.7982116 Flask Web开发：基于Python的Web应用开发实战（第2版）
+14 5.509352 Building Recommendation Systems in Python and JAX
+11 5.263399 OpenAI GPT For Python Developers, 2nd Edition
+13 4.4659142 Learn AI-Assisted Python Programming: With GitHub Copilot and ChatGPT
+12 1.8816761 Developing Apps with GPT-4 and ChatGPT
+4 1.5154111 白话深度学习的数学
+3 1.3515654 大模型应用开发极简入门
+7 1.2369337 Apache Pulsar实战
+
+query IFT
+SELECT id, score(), title FROM books WHERE match('title^5, description^1.2', 'ChatGPT') ORDER BY score() DESC
+----
+1 14.471097 这就是ChatGPT
+12 10.599274 Developing Apps with GPT-4 and ChatGPT
+13 7.9292374 Learn AI-Assisted Python Programming: With GitHub Copilot and ChatGPT
+3 1.77537 大模型应用开发极简入门
+
+query IFT
+SELECT id, score(), title FROM books WHERE match('title^5, description^1.2', '设计') ORDER BY score() DESC
+----
+9 14.486509 Vue.js设计与实现
+10 10.238626 前端架构设计
+8 9.061771 Rust程序设计（第2版）
+7 3.2078874 Apache Pulsar实战
+
+query IFT
+SELECT id, score(), title FROM books WHERE match(title, '设计 AND 实现') ORDER BY score() DESC
+----
+9 5.063791 Vue.js设计与实现
+
+query IFT
+SELECT id, score(), title FROM books WHERE match(title, '设计 OR 实现') ORDER BY score() DESC
+----
+9 5.063791 Vue.js设计与实现
+10 2.0477252 前端架构设计
+8 1.8123543 Rust程序设计（第2版）
+
+query IFT
+SELECT id, score(), title FROM books WHERE match(title, '设计 OR 实现 OR 实战') ORDER BY score() DESC
+----
+9 5.063791 Vue.js设计与实现
+7 2.189928 Apache Pulsar实战
+10 2.0477252 前端架构设计
+8 1.8123543 Rust程序设计（第2版）
+5 1.7138567 BERT基础教程：Transformer大模型实战
+6 1.2924166 Flask Web开发：基于Python的Web应用开发实战（第2版）
+
+query IFT
+SELECT id, score(), title FROM books WHERE match(title, '(设计 AND 实现) OR 实战') ORDER BY score() DESC
+----
+9 5.063791 Vue.js设计与实现
+7 2.189928 Apache Pulsar实战
+5 1.7138567 BERT基础教程：Transformer大模型实战
+6 1.2924166 Flask Web开发：基于Python的Web应用开发实战（第2版）
+
+query IFT
+SELECT id, score(), title FROM books WHERE match(title, 'Rust OR database') ORDER BY score() DESC
```

Comment:
If the keyword is `Rust OR database`, this syntax will not work? 
By the way, what is the`match` specification we are using?

Reply:
https://www.elastic.co/guide/en/elasticsearch/reference/current/sql-functions-search.html#sql-functions-search
The syntax of elastic search is referenced.